qiskit/circuit/quantumcircuit.py

```diff
@@ -545,14 +548,15 @@ def cbit_argument_conversion(self, clbit_representation):
         """
         return QuantumCircuit._bit_argument_conversion(clbit_representation, self.clbits)
 
-    def append(self, instruction, qargs=None, cargs=None):
+    def append(self, instruction, qargs=None, cargs=None, phase=0):
```

Comment:
i think we should take an approach similar to https://github.com/Qiskit/qiskit-terra/pull/4478 which does not modify append.

Reply:
Sounds good.

Reply:
```suggestion
    def append(self, instruction, qargs=None, cargs=None, phase=None):
    
        if phase is None:
            phase = 0
```

Reply:
This option is going away here.